ote/src/cljs/ote/style/base.cljs

```diff
@@ -13,8 +13,27 @@
 (def action-button-container (merge inline-block
                                     {:margin-right "1em"}))
 
-(def action-button {:padding-left "0.3em" :padding-right "0.3em"})
+(def action-button {:padding-left "1.1em"
+                    :padding-right "1.1em"
+                    :text-transform "uppercase"
+                    :color "#FFFFFF"
+                    :background-color "#1565C0"
+                    :font-size "12px"
+                    :font-weight "bold"
+                    })
 
+(def cancel-button {:padding-left "1.1em"
```

Comment:
move common style defs to `base-button` or something like that, and use it with `merge` for action and cancel buttons

Reply:
I made only one button style, base-button.